microprofile/WFLY-12737_quickstart-for-mp-health-2.2.adoc

```diff
@@ -0,0 +1,68 @@
+= WFLY-12737 Quickstart for MP Health 2.2
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile Health 2.2 use in WildFly /
+JBoss EAP. The example will cover all basic functionalities of MicroProfile Health 2.2,
+namely: liveness and readiness checks, the basic usage of the `HealthCheckResponseBuilder`,
+and the inclusion of user specific data in the health check responses.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-12737[WFLY-12737]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1368[EAP7-1368] (internal)
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* mailto:szhantem@redhat.com[Sultan Zhantemirov]
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* Quickstarts
+
+=== Other Interested Projects
+
+== Requirements
+
+=== Hard Requirements
+
+* include liveness health check
+* include readiness health check
+* include positive health check response
+* include negative health check response
+* include health check response with user specific data
+
+=== Non-Requirements
+
```

Comment:
One of the new features in MP Health 2.2 is that `HealthCheckResponse` [1] has been now defined as a concrete class. Maybe a mention to the fact that this QS is not demonstrating this very feature could be added here. 

[1]
https://download.eclipse.org/microprofile/microprofile-health-2.2/microprofile-health-spec.html#_apispi_changes